modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -26,32 +26,8 @@ class Configuration(object):
     :param api_key_prefix: Dict to store API prefix (e.g. Bearer)
     :param username: Username for HTTP basic authentication
     :param password: Password for HTTP basic authentication
-    :param key_id: The identifier of the cryptographic key, when signing HTTP requests.
-        An 'Authorization' header is calculated by creating a hash of select headers,
-        and optionally the body of the HTTP request, then signing the hash value using
-        a private key which is available to the client.
-    :param private_key_path: The path of the file containing a private key,
-        when signing HTTP requests.
-    :param signing_scheme: The signature scheme, when signing HTTP requests.
-        Supported value is hs2019.
-    :param signing_algorithm: The signature algorithm, when signing HTTP requests.
-        Supported values are PKCS1v15, PSS; fips-186-3, deterministic-rfc6979.
-    :param signature_max_validity: The signature max validity,
-        expressed as a datetime.timedelta value.
-    :param signed_headers: A list of strings. Each value is the name of a HTTP header
-        that must be included in the HTTP signature calculation.
-        The two special signature headers '(request-target)' and '(created)' SHOULD be
-        included in SignedHeaders.
-        The '(created)' header expresses when the signature was created.
-        The '(request-target)' header is a concatenation of the lowercased :method, an
-        ASCII space, and the :path pseudo-headers.
-        When signed_headers is not specified, the client defaults to a single value,
-        '(created)', in the list of HTTP headers.
-        When SignedHeaders contains the 'Digest' value, the client performs the
-        following operations:
-        1. Calculate a digest of request body, as specified in RFC3230, section 4.3.2.
-        2. Set the 'Digest' header in the request body.
-        3. Include the 'Digest' header and value in the HTTP signature.
+    :param signing_info: Configuration parameters for HTTP signature.
+        Must be an instance of {{{packageName}}}.HttpSigningConfiguration
```

Comment:
Can you update this to `{{{packageName}}}.signing.HttpSigningConfiguration`?

Reply:
done.